sdk/cosmosdb/azure_data_cosmos/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "azure_data_cosmos"
+version = "0.19.0"
+description = "Rust wrappers around Microsoft Azure REST APIs - Azure Cosmos DB"
+readme = "README.md"
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+rust-version.workspace = true
+homepage = "https://github.com/azure/azure-sdk-for-rust"
+documentation = "https://docs.rs/azure_data_cosmos"
+keywords = ["sdk", "azure", "rest", "cloud", "cosmosdb", "database"]
+categories = ["api-bindings"]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+async-trait.workspace = true
+azure_core.workspace = true
+typespec_client_core = { workspace = true, features = ["derive"] }
+time.workspace = true
+tracing.workspace = true
+url.workspace = true
```

Comment:
We re-export this, and should for `time` as well. Any reason not to reuse those? Ideally I was hoping to "hide" most dependencies such that `azure_core` is, I'd hope, the only one callers would need (apart from `serde` and a few others, of course).

Reply:
We need `form_urlencoded` from the `url` crate, which isn't re-exported (and I'd prefer we don't get too deep down the rabbit hole of re-exporting all the functionality from dependent crates out of `typespec_client_core`.

Reply:
I _can_ use the re-export of `OffsetDateTime` though.